app/assets/javascripts/components/option-select.js

```diff
@@ -238,9 +270,27 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     }
 
     // Resize to cut last item cleanly in half
-    var lastVisibleCheckbox = this.getVisibleCheckboxes().last()
-    var position = lastVisibleCheckbox.parent()[0].offsetTop // parent element is relative
-    this.setContainerHeight(position + (lastVisibleCheckbox.height() / 1.5))
+    var visibleCheckboxes = this.getVisibleCheckboxes()
+
+    var lastVisibleCheckbox = visibleCheckboxes[visibleCheckboxes.length - 1]
+    var position = $(lastVisibleCheckbox).parent()[0].offsetTop // parent element is relative
+    this.setContainerHeight(position + (this.getHeight(lastVisibleCheckbox) / 1.5))
+  }
+
+  OptionSelect.prototype.getHeight = function (el) {
+    if (el) {
+      return el.clientHeight
+    } else {
+      return 0
+    }
+  }
```

Comment:
Just setting a reminder to check this out in a browser, it may not work in IE8.

Reply:
[MDN say's it's been available in IE since IE5](https://developer.mozilla.org/en-US/docs/Web/API/Element/clientHeight) - though we don't support IE8 anymore 😉 

Are there many places where we expect that the element calling this not to exist? otherwise we don't really need this to be a separate function.